src/components/LeftSidebar/SearchConversationsResults/SearchConversationsResults.vue

```diff
@@ -0,0 +1,314 @@
+
+// Item's content (avatar) + internal_padding * 2 + external_padding * 2
+const itemSize = computed(() => isCompact.value ? 28 + 2 * 2 + 0 * 2 : AVATAR.SIZE.DEFAULT + 2 * 4 + 2 * 2)
+
+// Other sections than joined conversations in the output order
+const sections = [
+	{
+		type: 'user',
+		caption: { type: 'caption', id: 'users_caption', name: t('spreed', 'Users') },
+	},
+	{
+		type: 'group',
+		caption: { type: 'caption', id: 'groups_caption', name: t('spreed', 'Groups') },
+	},
+	{
+		type: 'circle',
+		caption: { type: 'caption', id: 'circles_caption', name: t('spreed', 'Teams') },
+	},
+	{
+		type: 'federated',
+		caption: { type: 'caption', id: 'federated_users_caption', name: t('spreed', 'Federated users') },
+	},
+] as const
+
+type subListType = {
+	user: ParticipantSearchResult[],
+	group: ParticipantSearchResult[],
+	circle: ParticipantSearchResult[],
+	federated: ParticipantSearchResult[],
+}
+
+const searchResultsVirtual = computed(() => {
+	// Initialize
+	const virtualList = []
+
+	const lowerSearchText = props.searchText.toLowerCase()
+	const searchResultsConversationList = props.conversationsList.filter(conversation =>
+		conversation.displayName.toLowerCase().includes(lowerSearchText)
+		|| conversation.name.toLowerCase().includes(lowerSearchText)
+	)
+
+	// Add conversations section
+	virtualList.push({ type: 'caption', id: 'conversations_caption', name: t('spreed', 'Conversations') })
+	if (searchResultsConversationList.length === 0) {
+		virtualList.push({ type: 'hint', id: 'hint_conversations', hint: t('spreed', 'No matches found') })
+	} else {
+		virtualList.push(...searchResultsConversationList.map((item) => ({ type: 'conversation', id: item.id, object: item })))
+	}
+
+	// Add "New Conversation" option if allowed
+	if (canStartConversations) {
+		virtualList.push({ type: 'listItem', id: 'new_conversation', name: props.searchText, subname: t('spreed', 'New private conversation') })
+	}
+
+	// Add open conversations section if any
+	if (props.searchResultsListedConversations.length !== 0) {
+		virtualList.push({ type: 'caption', id: 'open_conversation_caption', name: t('spreed', 'Open conversations') })
+		virtualList.push(...props.searchResultsListedConversations.map((item) => ({ type: 'open_conversation', id: item.id, object: item })))
+	}
+
+	// Categorize search results into different sections
+	const subList = props.searchResults.reduce((acc, conv) => {
+		if (conv.source === ATTENDEE.ACTOR_TYPE.USERS) {
+			acc.user.push(conv)
+		} else if (conv.source === ATTENDEE.ACTOR_TYPE.GROUPS && canStartConversations) {
+			acc.group.push(conv)
+		} else if (conv.source === ATTENDEE.ACTOR_TYPE.CIRCLES && canStartConversations) {
+			acc.circle.push(conv)
+		} else if (conv.source === ATTENDEE.ACTOR_TYPE.REMOTES && canStartConversations) {
+			acc.federated.push({ ...conv, source: ATTENDEE.ACTOR_TYPE.FEDERATED_USERS })
+		}
+		return acc
```

Comment:
`acc` is a common short form for `accumulator` but `conv` is not a common short name (and the list is no "conversations")

```suggestion
	const subList = props.searchResults.reduce((acc, result) => {
		if (result.source === ATTENDEE.ACTOR_TYPE.USERS) {
			acc.user.push(result)
		} else if (result.source === ATTENDEE.ACTOR_TYPE.GROUPS && canStartConversations) {
			acc.group.push(result)
		} else if (result.source === ATTENDEE.ACTOR_TYPE.CIRCLES && canStartConversations) {
			acc.circle.push(result)
		} else if (conv.source === ATTENDEE.ACTOR_TYPE.REMOTES && canStartConversations) {
			acc.federated.push({ ...result, source: ATTENDEE.ACTOR_TYPE.FEDERATED_USERS })
		}
		return acc
```